jak-na-konflikty.md

```diff
@@ -0,0 +1,77 @@
+# Jak zvládat konflikty s kolegou?
+
+Cílem této příručky je pomoct ti zvládat konfliktní situace mezi tebou a ostatními Appliftery.
+
+V Kodexu Appliftera se můžeš dočíst, že řešení konfliktů je přirozenou součástí každé organizace. Proto je důležité, abys věděl, jak ke konfliktu přistupovat tak, abys minimalizoval jeho toxicitu pro tebe a pro tvého kolegu a zároveň aby tvé jednání směřovalo k jeho vyřešení.
+
+## Příprava
+
+Před tím, než začneš s kolegou konflikt řešit, je dobré si v hlavě či na papíře ujasnit pár věcí. 
+
```

Comment:
Použil bych slovo "objektivně".

Reply:
@hauseja3 `..., je objektivně dobré si v hlavě...`? 😄 Kam bys to prsknul?

Reply:
Dal jsem to na blbej řádek. Třeba bych tím nahradil "odemočněný" předmět konfliktu.

Reply:
@hauseja3 Cool - když mně přijde, že to není moc o objektivitě, u konfliktu má každej svou pravdu, spíš o tom to vzít bez emocí.

Reply:
Já si myslím, že každý na to má svůj pohled a objektivní je velmi těžké mnohem důležitější je, aby to bylo bez emocí.

Reply:
Pro mě je to dost ekvivalentní, co se týče řešení konfliktů. Ale možná máš pravdu.